src/main/java/roomescape/service/ReservationService.java

```diff
@@ -16,19 +14,17 @@ public class ReservationService {
     private final ReservationRepository reservationRepository;
     private final ThemeRepository themeRepository;
 
-    public ReservationService(ReservationJdbcRepository reservationRepository, ThemeRepository themeRepository) {
+    public ReservationService(ReservationRepository reservationRepository, ThemeRepository themeRepository) {
         this.reservationRepository = reservationRepository;
         this.themeRepository = themeRepository;
     }
 
     public Long createReservation(ReservationRequestDto reservationRequest) {
         Theme theme = themeRepository
-                .findOneByName("워너고홈")
-                .orElseThrow(() -> {throw new NoSuchElementException("No Theme by that Name");});
-        if (reservationRepository.hasOneByDateAndTime(reservationRequest.getDate(), reservationRequest.getTime())) {
-            throw new IllegalArgumentException("Already have reservation at that date & time");
-        }
-        return reservationRepository.save(new Reservation(reservationRequest, theme));
+                .findOneById(reservationRequest.getThemeId())
+                .orElseThrow(() -> {throw new NoSuchElementException("No Theme by that Id");});
```

Comment:
Optional.orElseThrow() 를 사용할 때는 아래와 같이 사용해주시면 좋습니다 :)

```suggestion
                .orElseThrow(() -> new NoSuchElementException("No Theme by that Id"));
```